src/data/markdown/docs/30 xk6-browser/01 xk6-browser/09 Locator/selectoption--options--.md

```diff
@@ -0,0 +1,33 @@
+---
+title: 'selectOption(values, [options])'
+excerpt: 'xk6-browser: locator.selectOption method'
+---
+
+<Blockquote mod="warning">
+
+There are known issues with this feature. See [issue #470](https://github.com/grafana/xk6-browser/issues/470) and [issue #471](https://github.com/grafana/xk6-browser/issues/471) for details.
+
+</Blockquote>
+
+Select one or more options which match the values.
+
+| Parameter | Type                         | Description                                                                                                                                                                                                                    |
+|-----------|------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| values    | string or [string] or object | If the `select` has the multiple attribute, all matching options are selected, otherwise only the first option matching one of the passed options is selected. Object can be made up of keys with `value`, `label` or `index`. |
```

Comment:
WDYT about using TypeScript notation, so `string[]` instead of `[string]`?

Same below for the return type.

Reply:
Can we leave it as `[string]`? I've taken a quick look through our docs and it looks like we're using that notation for `arrays` e.g. http://localhost:8000/javascript-api/k6-http/options/

Reply:
I don't see it on that page :confused:

If you mean the notation in the method signature, that indicates that the argument is optional. It's not documenting the type.

From a quick grep, we only seem to be using that notation here and on the `30 xk6-browser.md` page. Since it's not a standard way to document a type, I think we should adopt TypeScript's `string[]` instead.

What we do on some other pages (e.g. [`httpx.batch()`](https://k6.io/docs/javascript-api/jslib/httpx/batch/)) is to document it as `array`. In some cases we might want to link to the page of the array element, but in this case `string[]` should be fine.

Reply:
Aha, i hadn't realised the difference. Thanks for clarifying -- I've changed it to `string[]`.